report-jtreg/src/main/resources/io/jenkins/plugins/report/jtreg/JenkinsReportJckGlobalConfig/help-comparatorArguments.html

```diff
@@ -20,20 +21,27 @@
 
         <li>"<code>%{S}</code>" or "<code>%{SPLIT}</code>" will be replaced with the spliterator specified in the field
             above.</li>
+
+        <li>"<code>%{dynamic}</code>", where you replace <code>dynamic</code> with a name of an item specified in the
+            <i>config items</i> section (specifically the name from the <i>Name of the item to find</i> field).
+            It will be replaced by a value from the corresponding config file from the current job and build.</li>
     </ul>
     <p>
         <i>Example of the macros:</i><br>
-        For the job name "<code>jtreg~full-jp11-ojdk11~rpms-f36.x86_64-fastdebug.sdk-f36.x86_64.testfarm-x11.defaultgc.legacy.lnxagent.jfroff</code>",<br>
-        and spliterator set to "<code>[.-]</code>", the macros "<code>%{2}%{S}%{N-1}_something_%{N}</code>" will be
-        replaced with "<code>jp11[.-]lnxagent_something_jfroff</code>".
+        There is a job with the job name "<code>jtreg~full-jp11-ojdk11~rpms-f36.x86_64-fastdebug.sdk-f36.x86_64.testfarm-x11.defaultgc.legacy.lnxagent.jfroff</code>",<br>
+        spliterator set to "<code>[.-]</code>" and a config item with the <i>Name of the item to find</i> field set
+        to "<code>nvr</code>" and its evaluation in the config file being "<code>java-17-openjdk-17.0.6.0.9-0.4.ea.el8</code>".
+    </p>
+    <p>
+        The macros "<code>%{2}%{S}%{N-1}_something_%{N} - %{nvr}</code>" will be
+        replaced with "<code>jp11[.-]lnxagent_something_jfroff - java-17-openjdk-17.0.6.0.9-0.4.ea.el8</code>".
     </p>
     <p>
         <i>So an example of the whole prompt with arguments can look like this:</i><br>
         <code>
             --compare<br>
             --history 5<br>
-            --build-config-find changelog.xml:nvr:/build/nvr<br>
-            --nvr ".*"<br>
+            --nvr "%{nvr}"<br>
```

Comment:
it maybe worthy to keep also the --nvr ".*" example. Otherwise users may fell to assumption, that  it is the final form, and no xtensions possible